specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulListTrainingJobs.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/{projectName}/train/jobs/{jobId}": {
+      "get": {
+        "description": "Gets the status for a training job.",
+        "operationId": "ConversationalAnalysisAuthoring_GetTrainingStatus",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "jobId",
+            "description": "The job ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The training job result.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringTrainingJobState"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Train Status": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetTrainStatus.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/{projectName}/train/jobs/{jobId}/:cancel": {
+      "post": {
+        "description": "Triggers a cancellation for a running training job.",
+        "operationId": "ConversationalAnalysisAuthoring_CancelTrainingJob",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "path",
+            "name": "jobId",
+            "description": "The job ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The job was successfully canceled."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Cancel Training Job": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulCancelTrainingJob.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/deletion-jobs/{jobId}": {
```

Comment:
Why does this follow a different pattern? Wouldn't ../delete/jobs/... be more consistent with other job status endpoints?

Reply:
Because there's no `delete` operation like the other ones.
- For `:train` we have `/train/{jobId}`.
- For `:export` we have `/export/{jobId}`.

You'll find that this is the same approach take in QnA.